Shop.BLL/Exceptions/InternalExceptions/InternalException.cs

```diff
@@ -0,0 +1,5 @@
+namespace Shop.BLL.Exceptions.InternalExceptions
+{
+    public class InternalException() :
+        Exception("Internal error occured during request processing");
```

Comment:
you need to pass a variable, and write the message itself in a series like string message